dlrs/main/classes/ManageLookupRollupSummariesNewController.cls

```diff
@@ -0,0 +1,364 @@
+        try {
+            // Delete the rollup custom metadata
+            RollupService.delete_X(new List<String> { LookupRollupSummary.DeveloperName });
+            // Reload this page (and thus the rollup list in a new request, metadata changes are not visible until this request ends)
+            PageReference newPage = Page.managelookuprollupsummaries;
+            newPage.setRedirect(true);
+            return newPage;                 
+        } catch (Exception e) {
+            ApexPages.addMessages(e);
+        }*/
+        return null;
+    }
+
+    public PageReference hideMessage() {
+        DeclarativeLookupRollupSummaries__c settings = DeclarativeLookupRollupSummaries__c.getOrgDefaults();
+        settings.HideManageLookupRollupSummariesInfo__c = true;
+        upsert settings;
+        return null;
+    }
+
+    public List<SelectOption> getCalculationModes(){
+        List<SelectOption> options = new List<SelectOption>();
+        for(Schema.Picklistentry picklistEntry : LookupRollupSummary__c.fields.CalculationMode__c.getDescribe().getPickListValues()) {
+            options.add(new SelectOption(picklistEntry.getValue(), picklistEntry.getLabel()));
+        }
+        return options;
+    }           
+
+    public List<SelectOption> getAggregateOperations(){
+        List<SelectOption> options = new List<SelectOption>();
+        for(Schema.Picklistentry picklistEntry : LookupRollupSummary__c.fields.AggregateOperation__c.getDescribe().getPickListValues()) {
+            options.add(new SelectOption(picklistEntry.getValue(), picklistEntry.getLabel()));
+        }
+        return options;
+    }           
+
+    public List<SelectOption> getCalculationSharingModes(){
+        List<SelectOption> options = new List<SelectOption>();
+        for(Schema.Picklistentry picklistEntry : LookupRollupSummary__c.fields.CalculationSharingMode__c.getDescribe().getPickListValues()) {
+            options.add(new SelectOption(picklistEntry.getValue(), picklistEntry.getLabel()));
+        }
+        return options;
+    }
+
+    public List<Selectoption> getParentObjList(){
+        objectDescription = Schema.getGlobalDescribe();
+        List<Schema.SObjectType> parentObjList = objectDescription.Values();
+        List<SelectOption> objNames = new List<SelectOption>();
+        objNames.add(new SelectOption('','-- Select --'));
+        for(Schema.SObjectType obj : parentObjList)
+        {
+            Schema.DescribeSObjectResult description = obj.getDescribe(SObjectDescribeOptions.DEFERRED);
+            String prefix = description.getKeyPrefix();
+            
+            if (!String.isEmpty(prefix)) {
+                String objName = description.getName();
+                objNames.add(new SelectOption(objName, obj.getDescribe().getLabel()+' ('+ objName +')'));
```

Comment:
Labels continued: getLabel() will return the localised or even overriden version of the object name. Hence we might have different languages being displayed in our picklists